app/konnect/mesh-manager/rbac.md

```diff
@@ -0,0 +1,54 @@
+---
+title: Mesh Manager RBAC
+content_type: reference
+---
+
+Mesh Manager has it's own role-based access control (RBAC) settings that are separate from the {{site.konnect_short_name}} RBAC settings. The Mesh Manager RBAC settings are specific to the meshes and mesh policies in Mesh Manager. 
+
+To completely configure RBAC for Mesh Manager, you must configure both roles and role bindings.
+* **Role:** Determines what resources a user or group has access to
+* **Role binding:** Determines which users or groups are assigned to a particular role
+
+<!-- Is the admin role included by default/automatically? -->
+
+## Roles description
```

Comment:
I think this whole section (Roles Description) can be deleted, the relevant differences for MinK are in the mappings, some comments below on that.

Reply:
We can hyperlink out to the Mesh docs though.